geopandas/geodataframe.py

```diff
@@ -547,15 +549,19 @@ def to_crs(self, crs=None, epsg=None, inplace=False):
 
         Parameters
         ----------
-        crs : pyproj.CRS, optional if `epsg` is specified
+        crs: pyproj.crs.CRS, optional if `epsg` is specified
```

Comment:
Can you leave this as pyproj.CRS ?

(same reason as above: if the recommended way to use this is `pyproj.CRS`, I prefer that the docs also use that code path in the text. Otherwise it's IMO only confusing)

Reply:
Either way is valid. But, consistency is a good thing.

Reply:
Addressed,